build.sbt

```diff
@@ -260,7 +260,8 @@ lazy val jetty = libraryProject("jetty")
   .settings(
     description := "Jetty implementation for http4s servers",
     libraryDependencies ++= Seq(
-      jettyServlet
+      jettyServlet,
+      jettyHttp2Server
```

Comment:
This introduces a new dependency that people won't use by default, which is not great, but I don't want to create another Jetty module, and optional dependencies are frustrating.  I think this is the least of all evils.